backend/tests/v5/unit/processors/teamspaces/projects/models/drawings/calibrations.test.js

```diff
@@ -294,21 +296,23 @@ const testAddCalibration = () => {
 		const drawing = generateRandomString();
 		const revision = generateRandomString();
 		const createdBy = generateRandomString();
-		const calibration = generateRandomObject();
-		const drawingData = generateRandomObject();
+		const calibration = {
+			...generateRandomObject(),
+			verticalRange: [0, 5],
+			units: 'm',
+		};
 
 		test('should add a new calibration', async () => {
 			CalibrationsModel.getCalibration.mockResolvedValueOnce(calibration);
 
-			await Calibrations.addCalibration(teamspace, project, drawing, revision, createdBy,
-				calibration, drawingData);
+			await Calibrations.addCalibration(teamspace, project, drawing, revision, createdBy, calibration);
 
 			expect(CalibrationsModel.addCalibration).toHaveBeenCalledTimes(1);
 			expect(CalibrationsModel.addCalibration).toHaveBeenCalledWith(teamspace, project, drawing, revision,
-				createdBy, calibration);
+				createdBy, deleteIfUndefined({ ...calibration, verticalRange: undefined, units: undefined }));
```

Comment:
similar here, use the spread operator assignments to make hte code more readable